src/people.rs

```diff
@@ -56,25 +64,32 @@ macro_rules! define_person_property {
                 _context: &$crate::context::Context,
                 _person_id: $crate::people::PersonId,
             ) -> Self::Value {
-                $default
+                let $context = _context;
+                let $person = _person_id;
+                $ty
```

Comment:
I'm not sure I understand the implications of taking a function and making it a block.

Reply:
I changed this to take a closure as an expression, and having a second macro for `define_person_property_with_default`